--- gulpfile.js	2017-03-14 21:49:31.000000000 +0900
+++ gulpfile.custom.js	2017-03-18 19:20:12.000000000 +0900
@@ -6,7 +6,11 @@
     autoprefixer = require('gulp-autoprefixer'),
     browserSync = require('browser-sync').create();
 
+var replace = require('gulp-replace');
+
+var GOEMON_LIVERELOAD_CODE = '  <script src="http://localhost:35730/livereload.js"></script>';
 var DEST = 'build/';
+var DEST_ASSETS = 'assets/';
 
 gulp.task('scripts', function() {
     return gulp.src([
@@ -57,4 +61,14 @@
 });
 
 // Default Task
-gulp.task('default', ['browser-sync', 'watch']);
\ No newline at end of file
+gulp.task('default', ['browser-sync', 'watch']);
+
+// Build resources for golang web app (Add livereload code, replace path for goemon)
+gulp.task('build', function() {
+    gulp.src(['./production/*.html'])
+        .pipe(replace('href="../', 'href="/'))
+        .pipe(replace('\</head\>', GOEMON_LIVERELOAD_CODE + '\n</head>'))
+        .pipe(gulp.dest('./' + DEST_ASSETS));
+    gulp.src('./vendors/**').pipe(gulp.dest('./' + DEST_ASSETS + 'vendors/'));
+    gulp.src('./build/**/*').pipe(gulp.dest('./' + DEST_ASSETS + 'build/'));
+});
